daemon/k8s_watcher.go

```diff
@@ -832,6 +833,11 @@ func getUniqPorts(svcPorts map[types.FEPortName]*types.FEPort) map[uint16]bool {
 }
 
 func (d *Daemon) delK8sSVCs(svc types.K8sServiceNamespace, svcInfo *types.K8sServiceInfo, se *types.K8sServiceEndpoint) error {
+	// If east-west load balancing is disabled, we should not sync(add or delete)
+	// K8s service to a cilium service.
```

Comment:
Per our offline discussion / my other comment, given that we still want headless services to be enabled(?), this flag should be renamed and the documentation for it updated accordingly. 

To confirm, we should only disable Cilium management of Kubernetes LoadBalancing services, not headless services @tgraf @nebril ?